javascript/javascript1/week3/travelTime.js

```diff
@@ -0,0 +1,19 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 465,
+};
+//console.log(travelInformation["speed"]);
+function travelInfo(travelInformation) {
```

Comment:
when the function returns something, give it the name `getSomething`, so here it will be `getTime` because it returns time